lib/simp/cli/commands/bootstrap.rb

```diff
@@ -386,17 +423,18 @@ def self.ensure_puppet_processes_stopped
   # Ensure the puppetserver is running ca on the specified port.
   # Used ensure the puppetserver service is running.
   def self.ensure_puppetserver_running(port = nil)
-    port ||= `puppet config print masterport`.chomp
+    port = Simp::Cli::Utils.puppet_info[:config]['masterport']
```

Comment:
Wasn't the original logic was correct?  Since this configuration will vary over time in the FOSS case, it must be determined each time.

Reply:
@lnemsick-simp Ah, got it. Forgot about that.